### ISSUE TITLE
Conditional independence test for mixed dataset (binary and continuous data)

### ISSUE THREAD

**Issue by asha24choudhary** [jakobrunge/tigramite#351]
Hi there. I was going through the conditional independence tests and found for non-linear dataset, the independence tests recommended is GDPC & CMIknn. However, if I have a dataset with mixed dataset which includes binary/ discrete variables and **non-linear** continuous variables, which independence test, would you recommend. I found the regressionCI conditional independence test for the mix dataset, however, the continuous data is linear. Could you please help me with this?

**Comment by jakobrunge** [comment]
We will have a test for this implemented in the next 1-2 weeks.

**Comment by jakobrunge** [comment]
CMIknnMixed will be out in the next days!

**Comment by manolotis** [comment]
> CMIknnMixed will be out in the next days!

Hi @jakobrunge, is there an update on this? Something like CMIknnMixed would be very helpful for me at the moment. 
I was searching through the documentation, and I saw that CMIknn and CMISysmb both have a reference to CMIknnMixed, which I did not find in tigramite/independence_tests. Digging a bit more, I saw there used to be one, which got removed last year in [this commit](https://github.com/jakobrunge/tigramite/commit/750d0969fa7bbe924f2e9ac6508d0ab97eb7aee7#diff-3db01f914c019b861878c694868504818635e57cecc3fd55243d4415e24c167f), right after it was updated in [this one](https://github.com/jakobrunge/tigramite/commit/3e5206309e786df56f5b23e28d3ca21753a7fbd5). Is there any reason it was removed, or if is it functional if I check out on that commit? 

Thank you in advance!
